File: CleanAddisBackOfficeApi/src/main/java/com/iyoa/cleanaddis/json/MediaJson.java
package com.iyoa.cleanaddis.json;

import java.util.UUID;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.iyoa.cleanaddis.model.Label;

import lombok.Data;

@Data
public class MediaJson {
	
	@JsonProperty("url")
	private String url;
	
	@JsonProperty("type")
	private String type;
	

	@JsonProperty("label")
	private String label;
	
	@JsonProperty("description")
	private String description;
	
}

File: app/src/main/java/com/iyoa/cleanaddis/data/common/MediaJSON.kt
package com.iyoa.cleanaddis.data.common

import androidx.room.ColumnInfo
import androidx.room.PrimaryKey
import com.iyoa.cleanaddis.data.posting.LabelUUID
import com.squareup.moshi.Json

data class MediaJSON (

    @Json(name="uuid")
    val uuid: String,

    @Json(name="url") val url:String,
    @Json(name="type")val type: String,
    @Json(name = "label") val label:LabelUUID,

    @Json(name="description")val description:String
){}